openflow15/openflow15.go

```diff
@@ -2581,3 +2586,8 @@ type Property interface {
 func (p *PropHeader) Header() *PropHeader {
 	return p
 }
+
+type OverFlowedMessage interface {
```

Comment:
It seems that only `PacketIn2PropPacket` implements `OverFlowedMessage` (i.e., has the `ActLength()` method). Given that I wonder if we really need the interface as we could replace `prop.(OverFlowedMessage)` with `prop.(*PacketIn2PropPacket)`?

Reply:
Sure, I would remove.